commands/project.go

```diff
@@ -71,15 +64,15 @@ func (pc *projectCmd) projectAction(ctx *cli.Context) error {
 		return cli.NewExitError("project list request failed "+err.Error(), 1)
 	}
 
-	// some juggling with the json return string
-	var resJSON interface{}
+	var resJSON []*Project
 	if err := json.Unmarshal(ret, &resJSON); err != nil {
 		return cli.NewExitError("failed to decode response", 1)
 	}
-	k := resJSON.([]interface{})
-	for _, v := range k {
-		x := v.(map[string]interface{})
-		fmt.Printf("Project %+v\n", x["title"])
+
+	t := template.New("project list template")
+	t, _ = t.Parse("Project : {{.Title}}  Guid : {{.Guid}} \n\n")
```

Comment:
This is looking good. One small comment here and a FYI you can use range in the template itself: 

```
{{range . }}
  "Project : {{.Title}}  Guid : {{.Guid}} \n\n"
{{end}}

```